addons/mail/static/src/components/message/message_tests.js

```diff
@@ -1502,6 +1502,44 @@ QUnit.test('rendering of tracked field of type many2one: from no related record
     );
 });
 
+QUnit.test('state should not be changed to "clicked" when author name of message is clicked', async function (assert) {
+    assert.expect(1);
+
+    await this.start();
+    const message = this.env.models['mail.message'].create({
+        author: [['insert', { id: 7, display_name: "Demo User" }]],
+        body: "<p>Test</p>",
+        id: 100,
+    });
+    await this.createMessageComponent(message);
+    document.querySelector(`.o_Message_authorName`).click();
+    await nextAnimationFrame();
+    assert.doesNotHaveClass(
+        document.querySelector(`.o_Message`),
+        'o-clicked',
+        "'o-clicked' should not be there when author name of message is clicked"
```

Comment:
As I commented before the move to message file, this assert comment should be more closely related to the functional aspect (similar to test title) rather than what is checked technically.

Reply:
"message should not be considered 'clicked' after clicking on its author name" or something like that should be good for both test title and assert comment